app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,44 @@
+# app/controllers/comments_controller.rb
```

Comment:
- [ ] Kindly remove this controller as it would have no matching routes after fixing the routes above.